browser_use/agent/views.py

```diff
@@ -58,6 +58,7 @@ class AgentSettings(BaseModel):
 	page_extraction_llm: Optional[BaseChatModel] = None
 	planner_llm: Optional[BaseChatModel] = None
 	planner_interval: int = 1  # Run planner every N steps
+	is_planner_reasoning: bool = False, # type: ignore
```

Comment:
```suggestion
	is_planner_reasoning: bool = False # type: ignore
```
unless you meant it to be a tuple

Reply:
Thanks! @pirate